### PR TITLE
Windows

### PR DESCRIPTION
**Pull request by xoviat**
Closes gh-73.

### CODE REVIEW

**Comment by xoviat** [comment]
@conda-forge-admin Please rerender.

**Comment by xoviat** [comment]
@conda-forge-admin Please rerender.

**Comment by xoviat** [comment]
Ping @ocefpaf. I don't think we need to wait for OSX.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
 build:
   number: 200
   # We lack openblas on Windows, and therefore can't build numpy there either currently.
```
Comment (basnijholt) [inline recipe/meta.yaml]: This isn't true anymore, right?

**Comment by basnijholt** [comment]
What is the difference with https://github.com/conda-forge/numpy-feedstock/pull/73 exactly?

**Comment by ocefpaf** [comment]
Let's try this! If we can always pull the binaries if we find any issues with this package. Hopefully if will integrate nicely in conda-forge.

Thanks @xoviat for your efforts to bring fortran depended packages to Windows!

**Comment by isuruf** [comment]
> What is the difference with #73 exactly?

This one builds numpy 1.14 and #73 built numpy 1.13 with some patches from 1.14 backported. We can build 1.13 later if there is demand and we encounter no issues with 1.14.

**Comment by ocefpaf** [comment]
> This one builds numpy 1.14 and #73 built numpy 1.13 with some patches from 1.14 backported. We can build 1.13 later if there is demand and we encounter no issues with 1.14.

I would be more interested in build `numpy 1.11` but I guess that is too much of a hassle to be worth it...

**Pull request merged**